docs/docs/guide/express-js.mdx

```diff
@@ -0,0 +1,168 @@
+---
+title: Express.js and GrowthBook
+description: This guide shows how to integrate GrowthBook with Express.js to manage server-side feature flags.
+slug: express-js
+---
+
+# Feature Flags with Express.js and GrowthBook
+
+This guide walks through how to add GrowthBook feature flags to an Express.js application. It assumes you are starting from scratch, so if you already have an Express.js application, you can skip to step 2.
+
+## 1. Create an Express.js app
+
+First, install Express.js
+
+```bash npm2yarn
+npm install --save express
```

Comment:
just fyi: Since npm v5, you don't need to include the `--save` flag, as modules are installed as deps by default.